<?xml version="1.0"?>

<!-- Oct 26, 2017 -->


            <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
			<!--                                                                                    -->
		    <!--                                                                                    -->
	        <!--          LAUNCH HUMAN EXPERT BASED EXPLORATION FOR GAS EMISSION MONITORING         -->
			<!--                                                                                    -->
			<!--                                                                                    -->
			<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
			<!--                              @AsifArain, Oct 26, 2017                              -->
			<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->

<launch>

	<node pkg="plan_execution" type="scan_publisher_node" name="scan_publisher" output="screen">
	
	    <param name="field_of_view"	value="270.0"/>
    	<param name="sensing_range"	value="15.0"/> <!-- FIXME 8.25 is 10 meter in real -->
	

    </node>
</launch>


<!-- NOTES:

rosservice call /ptu_control/sweep "{min_pan: -45.0, max_pan: 45.0, min_tilt: -10.0, max_tilt: -10.0, n_pan: 1, n_tilt: 1, samp_delay: 0.1}"

For sensing range 10m: tilt angle= -5.1655
For sensing range 15m: tilt angle= -3.45
For sensing range 30m: tilt angle= -1.73




-->
